- Description: qeth and lcs: qeth built as module cannot be loaded
  Symptom:     If qeth/lcs is built as a loadable module it cannot be loaded
               due to unresolved symbols.
  Problem:     net/ipv6/mcast.c does not export register_multicast6_notifier
               and unregister_multicast6_notifier.
  Solution:    Added missing EXPORT_SYMBOL statements.
  Problem-ID:  9632
  Archs:       s390-31, s390-64

--- linux-2.5/net/ipv6/mcast.c	20 Jun 2004 23:48:24 -0000	1.24.2.1
+++ linux-2.5/net/ipv6/mcast.c	25 Jun 2004 12:51:04 -0000	1.24.2.2
@@ -141,6 +141,8 @@
         return notifier_chain_unregister(&multicast_chain,nb);
 }
 
+EXPORT_SYMBOL(register_multicast6_notifier);
+EXPORT_SYMBOL(unregister_multicast6_notifier);
 
 static void igmp6_join_group(struct ifmcaddr6 *ma);
 static void igmp6_leave_group(struct ifmcaddr6 *ma);

